src/main/resources/GitHub_Light.xml

```diff
@@ -1,1090 +1,1099 @@
 <scheme name="GitHub Light" version="142" parent_scheme="Default">
-    <metaInfo>
```

Comment:
@napsterv update exact changes which you want to be changed.. don't include formatting changes.